bundle.yaml

```diff
@@ -78,7 +78,7 @@ spec:
                 description: Repository to pull the Hazelcast Platform image from.
                 type: string
               version:
-                default: '{page-latest-supported-hazelcast}-slim'
+                default: "5.0"
```

Comment:
Something to consider:

Moving the `bundle.yaml` to the root means we can't use attributes to keep the Platform version in sync with the rest of the docs. We would need to remember to update this field separately for each new release.

To be able to use these attributes, we could host the `bundle.yaml` in the `modules/ROOT/attachments` directory. This way the public URL to get the file will be `docs.hazelcast.com/hazelcast-operator/{version}/_attachments/bundle.yaml`

Reply:
Again, you are right. However, we are planning to update this `bundle.yaml` on each release of the docs with the newly generated one. Thus, we can keep it as it is for now.